test/readme.md

```diff
@@ -23,16 +23,16 @@ tests will be skipped.
 
 Below is a summary of the types of test automation in this repo.
 
```

Comment:
I have a markdown linter installed locally.  I figured the whitespace changes were fine.  I'm also happy to unformat the file or install a markdown plugin for eslint instead.